hr_worked_days_from_timesheet/__init__.py

```diff
@@ -18,8 +18,5 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from . import (
```

Comment:
Hi @andhit-r thanks for the contribution.
Can you make sure to shorten all your headers as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3